### PR TITLE
Update Auxiliary Function pages with new switches

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-auxiliary-functions.rst [hunk 1]
```diff
@@ -170,6 +170,26 @@ Supported Features
 +----------------------+----------------------------+----------+---------+---------+
 |        77            | TAKEOFF mode               |          |    X    |         |
 +----------------------+----------------------------+----------+---------+---------+
+|        78            | RunCam Control             |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        79            | RunCam OSD Control         |    X     |    X    |    X    |
++----------------------+----------------------------+----------+---------+---------+
+|        80            | VISODOM Calibrate          |    X     |         |         |
```
Comment (rmackay9) [inline common/source/docs/common-auxiliary-functions.rst:177]: sorry to be picky but can we chcnage VISODOM to "VisualOdometry" or "VisOdom"?

File: common/source/docs/common-auxiliary-functions.rst [hunk 2]
```diff
@@ -616,12 +645,84 @@ This three position switch determines if surface tracking via rangefinder is tow
...
+
+.. raw:: html
+
+   </td>
+   </tr>
+   <tr>
+   <td><strong>VISODOM Calibrate</strong></td>
+   <td>
+
+Calibrate VISODOM's camera attitude to vehicles.
```
Comment (rmackay9) [inline common/source/docs/common-auxiliary-functions.rst:681]: Let's change this one to "align camera's heading to vehicle heading"

File: common/source/docs/common-auxiliary-functions.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-auxiliary-functions.rst:177]: the meta data says Viso Align now
should match that

File: common/source/docs/common-auxiliary-functions.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-auxiliary-functions.rst:681]: no problem

**Comment by Hwurzburg** [comment]
this has been incorporated into upcoming features pages

**Pull request closed without merging**